src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -386,7 +386,13 @@ private void LogAuthResult(AuthenticationResult? authenticationResult)
             if (string.IsNullOrEmpty(mergedOptions.Instance))
             {
                 var mergedOptionsMonitor = _serviceProvider.GetService<IOptionsMonitor<JwtBearerOptions>>();
-                mergedOptionsMonitor?.Get(JwtBearerDefaults.AuthenticationScheme);
```

Comment:
I wonder why we had JwtBearerDefaults.AuthenticationScheme.
Probably because it was for a background process on in a web API, and it that case the default scheme was JwtBearerOptions. but indeed, it's better to use `authenticationScheme`

Reply:
Yes, that's what I remember.